index.html

```diff
@@ -9,13 +10,13 @@
 </head>
 <body>
     <section class="container">
-        <h2 class="title-colors">
+        <h1 class="title-colors">
```

Comment:
Fantastic change. Accessibility is a big part of writing good HTML and I'm glad you caught the heading order.

Reply:
Thanks! I was reading up on accessibility and saw an opportunity to fix my error!